packages/switch/package.json

```diff
@@ -63,7 +63,7 @@
         "tslib": "^2.0.0"
     },
     "devDependencies": {
-        "@spectrum-css/switch": "^1.0.23"
+        "@spectrum-css/switch": "^2.0.0-beta.0"
```

Comment:
With this approved, does that mean you'll move to a stable release here? @pfulton

Reply:
Yes, that's correct @Westbrook. I just merged and released, and we're now at `@spectrum-css/switch@2.0.0`